CWE-94-Python_darwin/747929791_majsoul_wrapper/sdk.py

```
@@ -0,0 +1,483 @@
+            flow = pickle.loads(server.get_items(tot, n).data)
+            for flow_msg in flow:
+                result = liqi.parse(flow_msg)
+                handler.parse(result)
+                tot += 1
+            history_msg = history_msg+flow
+            pickle.dump(history_msg, open(filename, 'wb'))
+        time.sleep(0.2)
+
+
+def replayWebSocket(filename='ws_dump.pkl'):
+    # 回放历史websocket报文，按顺序交由handler.parse
```

Ruff (0.13.2)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)